src/Libraries/CoreNodes/String.cs

```diff
@@ -13,14 +13,6 @@ namespace DSCore
     /// </summary>
     public static class String
     {
-        // It has been moved to String.FromObject UI node, which is compiled 
-        // to built-in function __ToStringFromObject().
-        [NodeObsolete("FromObjectObsolete", typeof(Properties.Resources))]
-        public static string FromObject(object obj)
```

Comment:
I wonder why the UI node doesn't use this function instead 🤔. I would think that this would be preferred to using the built-in function `__ToStringFromObject()`.